### PR TITLE
Adds suffix TDy to multiple functions

### LINKED ISSUE #19: To avoid possible name collisions prefix calls with TDy
As noted by @nocollier in a comment [here](https://github.com/TDycores-Project/TDycore/pull/17#discussion_r301195844).

### PR DESCRIPTION
**Pull request by bishtgautam**
Suffix `TDy` is added to multiple functions to avoid
possible name collisions.

Fixes #19

### CODE REVIEW

File: .travis/install-petsc.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-git clone https://bitbucket.org/petsc/petsc.git
+git clone https://gitlab.com/petsc/petsc.git
```
Comment (nocollier) [inline .travis/install-petsc.sh]: A number of these commits look the same as in #26, did I just take too long to merge and thus they are showing up?

File: include/tdycore.h [hunk 2]
```diff
@@ -97,7 +97,7 @@ PETSC_EXTERN PetscErrorCode TDyUpdateState(TDy tdy,PetscReal *P);
 PETSC_EXTERN PetscErrorCode Pullback(PetscScalar *K,PetscScalar *DFinv,
                                      PetscScalar *Kappa,PetscScalar J,PetscInt nn);
 PETSC_EXTERN PetscInt TDyGetNumberOfCellVertices(DM dm);
-PETSC_EXTERN PetscInt TDyGetNumberOfFaceVertices(DM dm);
+PETSC_EXTERN PetscInt TDyTDyGetNumberOfFaceVertices(DM dm);
```
Comment (nocollier) [inline include/tdycore.h]: double prefix not needed?

File: .travis/install-petsc.sh (hunk 1 above)
Comment (bishtgautam) [reply .travis/install-petsc.sh]: I will rebase this branch of the latest master.

File: include/tdycore.h (hunk 2 above)
Comment (bishtgautam) [reply include/tdycore.h]: I will fix this.

**Comment by nocollier** [comment]
Looks good to me, this is still marked draft, are you ready to merge?

**Comment by bishtgautam** [comment]
Ready to merge. Thanks.

**Pull request merged**